DigiMeSDK/GuestConsent/Classes/PreConsentViewController.m

```diff
@@ -26,22 +27,36 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
+    [self buildUiAndRotate:NO];
+}
+
+- (void)buildUiAndRotate:(BOOL)rotate
```

Comment:
`UI` stands for `UserInterface` therefore both `U` and `I` should be capitalised.